test/openshift/e2e/sequential/1-105_validate_label_selector/01-add-label-selector.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+# patches the subscription to add label selector in argocd-operator
+- script: |
+    oc set env deployment openshift-gitops-operator-controller-manager -n openshift-gitops-operator ARGOCD_LABEL_SELECTOR=foo=bar -c manager
```

Comment:
Please handle scenarios for both OLM based installation as well as non OLM installation. You can use an environment variable to switch between the 2 installation methods.
Check this file for reference : https://raw.githubusercontent.com/saumeya/gitops-operator/master/test/openshift/e2e/ignore-tests/sequential/1-018_validate_disable_default_instance/02-patch-subscription.yaml